libindy/src/api/anoncreds.rs

```diff
@@ -65,19 +63,18 @@ pub extern fn indy_issuer_create_and_store_claim_def(command_handle: i32,
 }
 
 /// Create a new revocation registry for the given claim definition.
-/// Stores it in a secure wallet identifying by the returned key.
+/// Stores it in a secure wallet.
 ///
 /// #Params
 /// wallet_handle: wallet handler (created by open_wallet).
 /// command_handle: command handle to map callback to user context.
 /// issuer_did: a DID of the issuer signing revoc_reg transaction to the Ledger
-/// schema_seq_no: seq no of a schema transaction in Ledger
+/// schema_json: schema as a json
```

Comment:
Why didn't we need it before?

Reply:
We used to use issuer_did:schema_seq_no to identify rev_reg inside wallet. Now we have following key issuer_did:(schema_key)